test/e2e/tests/resize_e2e_test.go

```diff
@@ -39,6 +39,7 @@ const (
 	defaultTier          = "STANDARD"
 	defaultNetwork       = "default"
 	minVolumeSize  int64 = 1 * util.Tb
+	defaultEpsilon int64 = 70 * util.Gb
```

Comment:
can you put a comment explaining some background reasoning of this number

Reply:
Done